app/src/ui/app-error.tsx

```diff
@@ -69,18 +65,7 @@ export class AppError extends React.Component<IAppErrorProps, IAppErrorState> {
   }
 
   private onDismissed = () => {
```

Comment:
Maybe this `onDismissed` can be removed entirely and just use `this.props.onDismissed` wherever is needed?